src/Microsoft.Extensions.Logging.Testing/Microsoft.Extensions.Logging.Testing.csproj

```diff
@@ -15,6 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
```

Comment:
It certainly could be. We could consider putting it in Common as a sources package.